model/accounts_mgmt/v1/subscription_type.model

```diff
@@ -79,6 +79,8 @@ class Subscription {
 	CloudProviderID String
 	RegionID String
 	ClusterBillingModel BillingModel
+	// Metrics array (max one element will present)
+	Metrics []OneMetric
```

Comment:
Why an array if it is going to have only one element?

Reply:
There are no optional items in openapi. So zero-length array will use `omitempty` and not be marshalled at all. And one-item array will be. So in openapi it is declared as an array. What should I do here?

Reply:
If you define it as an array it will look like this in the OpenAPI:

```json
"metrics": {
  "type": "array",
  "items": {
    "$ref": "#/components/schemas/ClusterMetrics"
  }
}
```

And the JSON generated/accepted by the SDK will look like this:

```json
{
  "kind": "Subscription",
  "id": "123",
  "metrics": [
    {
      "console_url": "http://my.example.com",
      "region": "myregion"
    }
  ]
}
```

If you define it as plain attribute instead it will look like this:

```json
"metrics": {
  "$ref": "#/components/schemas/ClusterMetrics"
}
```

The JSON generated/accepted by the SDK will look like this:

```json
{
  "kind": "Subscription",
  "id": "123",
  "metrics": {
    "console_url": "http://my.example.com",
    "region": "myregion"
  }
}
```

If the "metrics" attribute don't have a value it won't be marshaled, in both cases. So I think it is perfectly OK (and desirable) to avoid using the array. Am I missing something?

Reply:
It's too late to change it in AMS as it is an array for a couple of months already. So I tried to make SDK to looks the same.

Reply:
Too bad. Consider fixing that in AMS in the future. For now just rename "OneMetric" to "ClusterMetrics", please.

Reply:
Renamed